ooni/agent/agent.py

```diff
@@ -6,9 +6,21 @@
 from ooni.agent.scheduler import SchedulerService
 
 class AgentService(service.MultiService):
+    """Contain any element to manage agent services.
```

Comment:
This should read:

"The AgentService is responsible for running the ooniprobe web UI and the scheduled periodic tasks as a system daemon."